app/views/posts/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1 style="color:darkgreen;">Welcome to the Lets Pair Program!</h1>
```

Comment:
Eventually we should switch this out to define the styles in a file in `app/assets/stylesheets/` but we can do that later.